src/client/components/board.js

```diff
@@ -10,7 +10,13 @@ import { readableBytes } from "../helpers/utils";
 import Nav from "./nav";
 
 import io from "socket.io-client";
-const socket = io("localhost:" + document.location.port);
+
+const PORT =
+  process.env.NODE_ENV === "development" && process.env.DEV_PORT
+    ? process.env.DEV_PORT
+    : document.location.port;
```

Comment:
i don't see the reason of this change.

Reply:
In dev mode ```document.location.port ``` default as 3000 and it will break